R/tmb.R

```diff
@@ -484,6 +485,11 @@ h_mmrm_tmb <- function(formula,
     attr(data, which = "dataname"),
     toString(match.call()$data)
   )
+  fun_call$weights <- ifelse(
+    !is.null(attr(weights, which = "dataname")),
+    attr(weights, which = "dataname"),
+    deparse(match.call()$weights)
+  )
```

Comment:
```suggestion
  weights_str <- attr(weights, which = "dataname")
  if (!is.null(weights_str)) {
    fun_call$weights <- weights_str
  }
```
because if it is NULL then we don't need to add it

Reply:
I changed it back for now as otherwise got 2 failed tests:
![image](https://user-images.githubusercontent.com/68403409/188781544-e827b1de-ead9-45fd-a1ce-cf14cde1867e.png)

Reply:
ok I changed the condition for the printing slightly (checking for `symbol` instead of not `NULL`) and then it works